components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -2587,9 +2625,13 @@ impl Animatable for TransformList {
             }
         }
     }
+}
 
+impl ToAnimatedZero for TransformList {
     #[inline]
-    fn get_zero_value(&self) -> Result<Self, ()> { Ok(TransformList(None)) }
+    fn to_animated_zero(&self) -> Result<Self, ()> {
+        Ok(TransformList(None))
```

Comment:
Hmm... So we return ok for a `TransformList`, but not for any other kind of lists? huh.